### DISCUSSION TITLE
CVE-2021-44228 log4j zero-day exploit

### DISCUSSION THREAD

**Discussion by eric-hf** [nextgenhealthcare/connect#4899] [Q&A] [answered]
- [NIST: CVE-2021-44228](https://nvd.nist.gov/vuln/detail/CVE-2021-44228)
- [New zero-day exploit for log4j Java library is an enterprise nightmare](https://www.bleepingcomputer.com/news/security/new-zero-day-exploit-for-log4j-java-library-is-an-enterprise-nightmare/)

Is there a way to upgrade versions of `log4j` on existing Mirth Connect servers?
Is the only option upgrading our Mirth Connect version? If so, where can we see which log4j versions are associated with each Mirth Connect version?

**Comment by eric-hf** [comment]
Related to: @rangercej;s https://github.com/nextgenhealthcare/connect/discussions/4892
_(sorry, I just ran a search for the CVE # and didn't see this existing post)_

**Comment by pacmano1** [comment, marked as answer]
See https://github.com/nextgenhealthcare/connect/discussions/4892#discussioncomment-1789526
